tests/unittest/_torch/visual_gen/test_attention_perf.py

```
@@ -834,6 +853,191 @@ def test_fa4_cross_attn_quick(
+            seq_len,
+            head_dim,
+            backend="TRTLLM",
+            verbose=verbose,
+            sage_attention_config=sage_cfg,
+        )
+
+    # ------------------------------------------------------------------
+    # Quick / CI tests
+    # ------------------------------------------------------------------
+
+    @pytest.mark.parametrize("sage_cfg,cfg_id", zip(_SAGE_CONFIGS, _SAGE_CONFIG_IDS))
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)